api/test/conftest.py

```diff
@@ -1,11 +1,34 @@
+import asyncio
+
 import pytest
-from asgiref.sync import async_to_sync
 
 from conf.asgi import application
 
 
+@pytest.fixture
+def get_new_loop():
+    loops: list[asyncio.AbstractEventLoop] = []
+
+    def _get_new_loop() -> asyncio.AbstractEventLoop:
+        loop = asyncio.new_event_loop()
+        loops.append(loop)
+        return loop
+
+    yield _get_new_loop
+
+    for loop in loops:
+        loop.close()
+
+
+@pytest.fixture(scope="session")
+def session_loop() -> asyncio.AbstractEventLoop:
+    loop = asyncio.new_event_loop()
+    yield loop
+    loop.close()
```

Comment:
This can't use `get_new_loop` because `get_new_loop` is intentionally function scoped, whereas the session loop, as its name implies, is scoped to the session, and therefore cannot depend on the function scoped fixture.